controllers/admin/AdminShopController.php

```diff
@@ -409,6 +409,16 @@ public function renderForm()
                 $group_desc = $this->trans('You can only move your shop to a shop group with all "share" options disabled -- or to a shop group with no customers/orders.', [], 'Admin.Shopparameters.Notification');
             }
 
+            $this->fields_form['input'][] = [
+                'type' => 'color',
+                'label' => $this->trans('Color', [], 'Admin.Catalog.Feature'),
+                'name' => 'color',
+                'desc' => [
+                    $this->trans('It will apply to the multistore header to underline your shop context.', [], 'Admin.Shopparameters.Feature'),
```

Comment:
```suggestion
                    $this->trans('It will only be applied to the multistore header to highlight your shop context.', [], 'Admin.Shopparameters.Feature'),
```